src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+import {
+  LinearProgressBar as CoreProgressBar,
+  LinearProgressBarProps,
+} from 'wix-ui-core/linear-progress-bar';
+
+import { TPAComponentProps } from '../../types';
+
+import { st, classes } from './ProgressBar.st.css';
+
+export interface ProgressBarProps
+  extends TPAComponentProps,
+    LinearProgressBarProps {
+  className?: string;
+  'data-hook'?: string;
```

Comment:
These props don't need to be declared, since they already come with the `TPAComponentProps`